lib/main.coffee

```diff
@@ -12,6 +12,9 @@ module.exports =
     wrapSelectionsInBrackets:
       type: 'boolean'
       default: true
+    showInGutter:
+      type: 'boolean'
+      default: true
```

Comment:
This should probably default to false for backwards compatibility and to make sure nothing regresses